crates/runtime/src/instance/allocator.rs

```diff
@@ -647,6 +666,7 @@ impl OnDemandInstanceAllocator {
         &self,
         module: &Module,
         store: &mut StorePtr,
+        memory_source: &MemorySource,
```

Comment:
why not just pass by value?

Reply:
A remnant from before when I had a different implementation where `MemorySource` was heavier so it wasn't `Copy`.

Fixed!